docs/reference.md

```diff
@@ -0,0 +1,9 @@
+# Reference
```

Comment:
is there a way to include all functions here automatically? Or it's manual?

Reply:
its manual so we have the ability to exclude, annotate etc. as not all functions are intended to be public, not everything is required to be documented. its up to us!